search-app/src/cmr/search/api/concepts_search.clj

```diff
@@ -33,7 +33,7 @@
 (defn- concept-type-path-w-extension->concept-type
   "Parses the concept type and extension (\"granules.echo10\") into the concept type"
   [concept-type-w-extension]
-  (-> #"^([^s]+)s(?:\..+)?"
```

Comment:
Any idea what could break by this change? It looks fine to me, but why would we specifically avoid strings with the letter 's' except as the last letter. Madness.

Reply:
This regex is what caused CMR-4586. It doesn't work with `services` which start with an "s".